### ISSUE TITLE
pair loss

### ISSUE THREAD

**Issue by HaoLiuHust** [seasonSH/DocFace#26]
seems the mps loss described in the paper is not the same with the code, which one is the right one?

**Comment by HaoLiuHust** [comment]
理解错了tf.reduce_max的意思···
